spec/css-spec.coffee

```diff
@@ -84,16 +84,41 @@ describe 'CSS grammar', ->
         expect(tokens[0]).toEqual value: '.- ', scopes: ['source.css', 'meta.selector.css']
 
     describe 'id selectors', ->
-      it 'tokenizes a standalone id selector', ->
-        {tokens} = grammar.tokenizeLine '#ch-1 {}'
+      it 'tokenizes \\#unicorn as id selector', ->
```

Comment:
Ok.  So the real problem is that if this fails, then the spec output is `it 'tokenizes \#unicorn as id selector'`.  Would you be ok with generalizing these, ie `it 'tokenizes common id selectors'`, `it 'tokenizes unicode id selectors'`, etc?